cmd/regex_format.go

```diff
@@ -326,16 +326,48 @@ func findUpperCaseOnIgnoreCaseFlag(lines []string, iFlag bool) (bool, string) {
 	// check if the file contains uppercase letters
 	if iFlag {
 		for number, line := range lines {
-			// ignore line if it starts with ##!
-			if strings.HasPrefix(line, "##!") {
+			// ignore line if it starts with the following regex: \s+##!
+			if regex.CommentRegex.MatchString(line) {
 				continue
 			}
-			found := strings.IndexAny(line, "ABCDEFGHIJKLMNOPQRSTUVWXYZ")
+			found := findUppercaseNonEscaped(line)
 			if found > -1 {
+				// show the column where the uppercase letter was found
+				// for a better visual match, we add equal symbols a and a caret in a line below
+				fill := ""
+				if found > 0 {
+					fill = strings.Repeat("=", found-1)
```

Comment:
Index is off by one if found > 0.
```suggestion
					fill = strings.Repeat("=", found)
```

Reply:
No, because the last char is the caret "pointing" up.

Reply:
But it is, check your output :) With `0`, `fill` will be empty. With `1`, `fill` will be `=`, which means that _position_ `0` will have `=`. So the "-1" is already implied by the logic of creating the fill string.

Reply:
Sorry, didn't realised you had changed it.